quiz/backend/stats/admin.py

```diff
@@ -18,18 +18,27 @@ class BadgeAdminConfig(admin.ModelAdmin):
         "points",
         "created_at",
     )
-    list_display_links = (
-        "name",
-    )
-    search_fields = (
-        "name",
-    )
-    list_editable = (
-        "points",
-    )
+    list_display_links = ("name",)
+    search_fields = ("name",)
+    list_editable = ("points",)
     save_on_top = True
     list_filter = ("points",)
     ordering = ("points",)
 
 
+class PlayerBadgeAdminConfig(admin.ModelAdmin):
+    list_display = (
+        "badge",
+        "player",
+        "obtained_on",
+    )
+    search_fields = (
+        "badge",
+        "player",
+    )
+    list_filter = ("obtained_on",)
+    ordering = ("obtained_on",)
```

Comment:
I would consider ordering "-obtained_on" so that the latest show first.

Reply:
done

Reply:
resolved